src/encoded/static/components/antibody.js

```diff
@@ -67,18 +67,28 @@ const LotComponent = (props, reactContext) => {
     const geneTips = [];
     targetKeys.forEach((key, i) => {
         const scientificName = targets[key].organism.scientific_name;
-        const geneName = targets[key].gene_name;
+        const geneName = targets[key].genes.map(
+            gene => gene.symbol
+        );
 
         // Add to the information on organisms from the targets
         organismComponents.push(<span key={key}>{i > 0 ? <span> + <i>{scientificName}</i></span> : <i>{scientificName}</i>}</span>);
         organismTerms.push(`targets.organism.scientific_name=${scientificName}`);
         organismTips.push(scientificName);
 
         // Add to the information on gene names from the targets
-        if (geneName && geneName !== 'unknown') {
-            geneComponents.push(<span key={key}>{i > 0 ? <span> + {geneName}</span> : <span>{geneName}</span>}</span>);
-            geneTerms.push(`targets.gene_name=${geneName}`);
-            geneTips.push(geneName);
+        if (geneName.length) {
+            geneName.map(
+                symbol => geneComponents.push(
+                    <span key={key}>{geneComponents.length > 0 ? <span> + {symbol}</span> : <span>{symbol}</span>}</span>
+                )
+            );
+            geneTerms.push(
+                ...geneName.map(
+                    symbol => `targets.genes.symbol=${symbol}`
+                )
+            );
```

Comment:
Similar thing here. I _think_ you can say geneTerms = geneName.map(…) if I’m understanding correctly.

Reply:
Actually, I’m not clear you need to clone geneName. Can you just use it directly below instead of cloning to geneTips?